packages/mdc-top-app-bar/_variables.scss

```diff
@@ -0,0 +1,32 @@
+//
+// Copyright 2017 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+$mdc-top-app-bar-row-height: 64px;
+$mdc-top-app-bar-icon-vertical-padding: 8px;
+$mdc-top-app-bar-icon-horizontal-padding: 24px;
+$mdc-top-app-bar-title-left-padding: 8px;
+$mdc-top-app-bar-section-horizontal-padding: 12px;
+
+$mdc-top-app-bar-icon-padding: 12px;
+
+$mdc-top-app-bar-mobile-breakpoint: 599px;
+
+$mdc-top-app-bar-mobile-row-height: 56px;
+$mdc-top-app-bar-mobile-icon-vertical-padding: 4px;
+$mdc-top-app-bar-mobile-icon-horizontal-padding: 24px;
```

Comment:
If we've determined that this will be the same, should we just remove it since it shouldn't need to be overridden from one breakpoint to another? Ditto title-left-padding.

Reply:
I'll update these to match the spec. They got a little out of wack while I was trying out a couple of things.